subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -173,8 +173,15 @@ enum node_rx_type {
 	NODE_RX_TYPE_MESH_ADV_CPLT = 0x13,
 	NODE_RX_TYPE_MESH_REPORT = 0x14,
 #endif /* CONFIG_BT_HCI_MESH_EXT */
-};
 
+/* Following proprietary defines must be at end of enum range */
+#if defined(CONFIG_BT_CTLR_USER_EXT)
+	NODE_RX_TYPE_PROPRIETARY_START = 0x15,
```

Comment:
Can we be consistent and use `NODE_RX_TYPE_USER_START` and `USER_END`?

Reply:
Implemented suggested change